README.adoc

```diff
@@ -1,17 +1,22 @@
+:projectKey: optaweb-employee-rostering
+:sonarBadge: image:https://sonarcloud.io/api/project_badges/measure?project={projectKey}
+:sonarLink: link="https://sonarcloud.io/dashboard?id={projectKey}"
+
 = Employee Rostering Optimization as a Service
 
-image:https://sonarcloud.io/api/project_badges/measure?project=optaweb-employee-rostering&metric=alert_status[
-"Quality Gate Status", link="https://sonarcloud.io/dashboard?id=optaweb-employee-rostering"]
-image:https://sonarcloud.io/api/project_badges/measure?project=optaweb-employee-rostering&metric=reliability_rating[
-"Reliability Rating", link="https://sonarcloud.io/dashboard?id=optaweb-employee-rostering"]
-image:https://sonarcloud.io/api/project_badges/measure?project=optaweb-employee-rostering&metric=security_rating[
-"Security Rating", link="https://sonarcloud.io/dashboard?id=optaweb-employee-rostering"]
-image:https://sonarcloud.io/api/project_badges/measure?project=optaweb-employee-rostering&metric=sqale_rating[
-"Maintainability Rating", link="https://sonarcloud.io/dashboard?id=optaweb-employee-rostering"]
-image:https://sonarcloud.io/api/project_badges/measure?project=optaweb-employee-rostering&metric=ncloc[
-"Lines of Code", link="https://sonarcloud.io/dashboard?id=optaweb-employee-rostering"]
-image:https://sonarcloud.io/api/project_badges/measure?project=optaweb-employee-rostering&metric=coverage[
-"Coverage", link="https://sonarcloud.io/dashboard?id=optaweb-employee-rostering"]
+image:https://travis-ci.com/kiegroup/optaweb-employee-rostering.svg?branch=master[
+"Build Status", link="https://travis-ci.com/kiegroup/optaweb-vehicle-routing"]
```

Comment:
Wrong link (should link to optaweb-employee-rostering, not optaweb-vehicle-routing)

Reply:
Copy-paste error. Thanks for the catch!